tailblocks/src/components/Teams/Members.js

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+
+const Members = () => {
+  return (
+    <section class='text-gray-700 body-font bg-gray-200'>
+      <div class='container px-5 py-24 mx-auto'>
+        <div class='flex flex-col text-center w-full mb-20'>
+          <h1 class='text-2xl font-medium title-font mb-4 text-gray-900'>
+            The real building blocks
+          </h1>
+        </div>
+        <div class='flex flex-wrap -m-4'>
+          <div class='p-4 lg:w-1/4 md:w-1/2'>
+            <div class='h-full flex flex-col items-center text-center'>
+              <img
+                alt='team'
+                class='flex-shrink-0 rounded-lg w-full h-56 object-cover object-center mb-4'
+                src='https://dummyimage.com/203x203'
+              />
+              <div class='w-full'>
```

Comment:
Shall we maintain the team details in a separate config file and import and iterate it here just to reduce the file size.

Reply:
created a new config file that contains an array of objects and mapped in the Team section.